server/lib/prune-newick.js

```diff
@@ -24,16 +25,24 @@ function pruneOutliers(newick, alignedFasta) {
 		}
 	}
 
-	// Compute average and standard deviation of all the pairs of distances
+	// Compute and store distances between each pair
 	getLeaves(newick)
-	let average = 0
-	let totalDistances = []
 	let distCache = {}
+	let geneLength = {}
 
 	for (let i = 0; i < leafNodes.length; i++) {
 		let node = leafNodes[i]
 		let species1 = node.ident ? node.name + '__' + node.ident : node.name
 		distCache[i] = {}
+		// Compute actual gene length
+		geneLength[i] = 0
+		for (let k = 0; k < sequenceMap[species1].length; k++) {
+			let seq = sequenceMap[species1]
+			if (seq[k] !== '-') {
+				geneLength[i]++
```

Comment:
Please refrain from the prefix/postfix unary operators; use `+= 1` instead

Reply:
Why?

Reply:
I personally detest them. Thy don't look like anything else in the language, they're not flexible, and I spend so little time seeing them that I have to do a double-take to understand them in any position outside of a c-style for-loop. 

So: personal preference. If you really want to use them, I can back down; I hesitated for like five minutes before I posted this request because it seems subjective, but I think there's only one other place in the codebase that uses this style (and that's not in a for-loop), and that's a few dozen lines below here in this file, so I may be able to make a "consistency" argument as well. 

---

I realize I'm in the weaker position since I'm asking you to change the code that you've already written, but I'll ask you this: why use `++` at all, outside of a c-style for-loop?

Reply:
Detest is the wrong word. I dislike them. I feel that `+= 1` is a much more expressive way to accomplish the same end result, is all.